config/locales/en.yml

```diff
@@ -40,7 +40,7 @@ en:
       status:
         hint:
           html: IATI activity status. <a href="http://reference.iatistandard.org/203/codelists/ActivityStatus/" target="_blank">See IATI for detailed descriptions.</a>
-      submit: Submit
+      submit: Continue
```

Comment:
I would prefer it if the key was `continue` instead of `submit`

Reply:
I don't agree - I'd much rather have consistency in the I18n keys for submit buttons across forms, i.e. `form.activity.submit` / `form.organisation.submit` etc. 

Plus, the new form builder uses the standard ActiveRecord I18n keys, and this generally sticks with `submit`.

Reply:
Fair comment!